modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -135,3 +135,16 @@
 
 	var/obj/item/organ/tongue/dog/new_tongue = new(get_turf(human_holder))
 	new_tongue.Insert(human_holder)
+
+/datum/quirk/stammer
+	name = "Stammer"
+	desc = "You stammer (stutter) whenever you speak"
+	icon = "comment-slash"
+	value = 0
+	medical_record_text = "Patient has an uncurable stammer, causing pauses and repeating of letters in their speech"
```

Comment:
```suggestion
	medical_record_text = "Patient has an uncurable stammer, causing pauses and repeating of letters in their speech."
```

Reply:
i'd change uncurable stammer to uncurable stutter, it sounds better